frontend/src/components/uam/common/enums/user-table-accesson.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum UserTableAccesson {
```

Comment:
same typo in file name, but I'm not sure if this enum is necessary
```suggestion
enum UserTableAccessor {
```